plugins/inputs/procstat/win_service_windows.go

```diff
@@ -39,6 +39,7 @@ func queryPidWithWinServiceName(winServiceName string) (uint32, error) {
 	}
 
 	buf = make([]byte, bytesNeeded)
+	//nolint:gosec // G103: Use of unsafe calls should be audited
 	p = (*windows.SERVICE_STATUS_PROCESS)(unsafe.Pointer(&buf[0]))
```

Comment:
I'm not sure if this is the best way to silence the warnings. Nobody will ever look at that spot again IMO. So either we should keep the linter issue here or we be sure that this is a valid use and silence it..

Reply:
@srebhan Yeah, that how it was done in first commit, but I was asked to removed it ;)
https://github.com/influxdata/telegraf/pull/13038#discussion_r1158469794

Reply:
@srebhan Check current version :)